### ISSUE TITLE
"Transaction reverted by EVM" in console, but success in UI

### ISSUE THREAD

**Issue by Fang-** [urbit/bridge#200]
I tried setting the voting proxy for a fakenet ~zod. Turns out the gas limit constant there doesn't account for Mysterious Overhead, and the tx reverts with out of gas. (In other words, you can reproduce this by setting a way-too-low gas limit.)

<img width="529" alt="image" src="https://user-images.githubusercontent.com/3829764/61539585-1e064d80-aa3c-11e9-9024-788c00c16f69.png">

It pretends everything's fine, but the "new value" presented is actually the old one, and my transaction failed under the hood.

**Comment by Fang-** [comment]
Can also reproduce with "issue child" from a fresh zod.

**Comment by shrugs** [comment]
Yup — errors from rejections don't seem to be caught by our trycatch in useEthereumTransaction. are we missing an `await` somewhere, or is web3 missing an await somewhere so that error is throw to nobody? really confusing experience for users that don't have a javascript console open
